README.adoc

```diff
@@ -0,0 +1,56 @@
+= htsjdk-next-beta
+:toc: macro
+
+image:http://img.shields.io/badge/language-java-brightgreen.svg[Language, link=https://www.java.com]
```

Comment:
If we go with asciidoc we might want a footnote at the bottom with a link to an asciidoc quick reference guide.